bot/iiwb/client/core.py

```diff
@@ -0,0 +1,56 @@
+from discord.ext import commands
+from discord.utils import get
+from iiwb.core._models import Context
+from iiwb.core import utils
+import asyncio
+import sys
+import json
+
+class Core(commands.Cog):
+
+	def __init__(self, bot):
+		self.bot = bot
+		self.cogs = utils.listCogs().keys()
+		self.defaultCogs = ['reverse.client.default', 'reverse.client.debugger.debugger']
+
+	@commands.command()
+	async def hey(self, ctx):
+		await ctx.send("Hello!")
+
+	@commands.command()
+	async def reload(self, ctx, *args):
```

Comment:
On my end, it seems to need 2 reloads before I can see the changes. idk why

Reply:
hmmmmmmmmm, show me on discord